### PR TITLE
Category graphql/searchable sort multiple fields

### PR DESCRIPTION
**Pull request by sundersc**
#### Description of changes
This change allows to use ElasticSearch sort with multiple fields.
[Link to Searchable RFC Document](https://github.com/aws-amplify/amplify-cli/issues/7546)

#### Issue #, if available
NA

#### Description of how you validated changes
- Manual testing
- yarn test passed

#### Checklist
- [x] PR description included
- [x] `yarn test` passes
- [x] Tests are [changed or added](https://github.com/aws-amplify/amplify-cli/blob/master/CONTRIBUTING.md#tests)
- [ ] Relevant documentation is changed or added (and PR referenced)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/amplify-graphql-searchable-transformer/src/__tests__/__snapshots__/amplify-graphql-searchable-transformer.tests.ts.snap [hunk 1]
```diff
@@ -97,12 +97,12 @@ input SearchablePostSortInput {
 
 type SearchablePostConnection {
   items: [Post]
-  nextToken: String
+  nextToken: [String]
```
Comment (cjihrig) [inline packages/amplify-graphql-searchable-transformer/src/__tests__/__snapshots__/amplify-graphql-searchable-transformer.tests.ts.snap:100]: I think the plan was to keep `nextToken` as a string. I think we can use the VTL functions for encoding and decoding to do this transparently.

Comment (sundersc) [reply packages/amplify-graphql-searchable-transformer/src/__tests__/__snapshots__/amplify-graphql-searchable-transformer.tests.ts.snap]: Updated to base64 string.

**Review by cjihrig — APPROVED** [review]

**Comment by yaquawa** [comment]
Hi @sundersc, is this for Transformer v1 ?

**Comment by sundersc** [comment]
> Hi @sundersc, is this for Transformer v1 ?

Hey @yaquawa this is for Transformer v2.

**Comment by yaquawa** [comment]
Thanks @sundersc ! Will this be implemented for Transformer  V1 also? Because at the moment I feel like I have to wait for years for V2 to be released.

**Pull request merged**